index.html

```diff
@@ -59,29 +59,29 @@ <h2 class="main_program">Main program</h2>
          </div>      
         <div class="guest_list">
             <div class="program" href="">
-                <img src="images/images.png" alt="" width="100" height="40">
+                <div class="program_images"><img src="images/pngwing.com.png" alt="" width="20" height="20"></div>
                 <h3>Lectures</h3>
-                <p class="explain">Talking about film</p>
+                <p class="explain">Talking about film and other things that is around such as music and casting actors </p>
             </div>
             <div class="program2" href="">
-                <img src="images/images (2).png" alt="icon" width="30" height="40">
-                <h3>Chat</h3>
-                <p class="explain">Get to know people</p>
+                <div class="program_images"><img src="images/images (2).png" alt="icon" width="30" height="40"></div>
+                <h3>Chatting</h3>
+                <p class="explain">Get to know people, getting to know people in a space that you have an interest</p>
             </div>
             <div class="program3">
-                <img src="images/—Pngtree—photo camera icon design _4173568.png" alt="images of camera icon" width="30" height="30">
+                <div class="program_images"><img src="images/—Pngtree—photo camera icon design _4173568.png" alt="images of camera icon" width="30" height="30"></div>
                 <h3>FORUM</h3>
-                <p class="explain">Shops will be open</p>
+                <p class="explain">Shops will be open, the shops is open to buy merchandise and other film that we recommend </p>
             </div>
             <div class="program4">
-                <img src="images/images (1).png" alt="" width="30" height="40">
-                <h3>Film</h3>
-                <p class="explain">watch a film</p>
+                <div class="program_images"><img src="images/images (1).png" alt="" width="30" height="40"></div>
+                <h3>Cinema</h3>
+                <p class="explain">watch a film, there will be theatre and it will be one at certain times </p>
             </div>
             <div class="program5" href="">
-                <img src="images/video-icon-8026.png" alt="" srcset="" width="40" height="40">
+                <div class="program_images"><img src="images/video-icon-8026.png" alt="" srcset="" width="40" height="40"></div>
                 <h3>Signings</h3>
-                <p class="explain">Get an autograph </p>
+                <p class="explain">Get an autograph, there will be other directors, casting, voice actors and film composers</p>
```

Comment:
- [ ] Kindly fix this as required in the previous [review](https://github.com/LethaboMot/capstone-project/pull/1/files#r1169321124), you should match your design to Figma one :wink: